### DISCUSSION TITLE
I want to contribute to Github Student Developer Pack with my startup

### DISCUSSION THREAD

**Discussion by savrum** [community/community#106549] [GitHub Education] [answered]
Hello, right now I'm working on a really simple e-commerce startup and I'm willing to let Github student pack users use it for completely free :) I don't know if there are any requirements needs to be filled to contribute to Student Developer Pack.

Are there any guides?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @savrum , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by savrum** [reply]
Thank you so much, I'll be more careful next time :)

**Reply by elonerajeev** [reply]
Hey @savrum  I'm currently working on #LearnTube Software , an awesome software for students (#####), and I'm eager to contribute it to the GitHub Student Developer Pack for free use. Does anyone know if there are specific requirements or guides for this? Thanks a bunch!

**Comment by elonerajeev** [comment]
Hey @savrum  I'm currently working on #LearnTube Software , an awesome software for students (#####), and I'm eager to contribute it to the GitHub Student Developer Pack for free use. Does anyone know if there are specific requirements or guides for this? Thanks a bunch!

**Comment by samus-aran** [comment, marked as answer]
Hi @savrum & @elonerajeev 

Student Developer Pack partners are an essential part of our student’s learning experience. We seek out and onboard 5-10 new partners per year. Partner tools and resources must align with the developer lifecycle or related fields and be offered at no cost to the student. Please fill out [this form](https://survey3.medallia.com/?IqFRAo-edu-partnership) in order to have your company considered. Please note that not all who inquire are approved to join the Student Developer Pack.

Good Luck 👍

**Reply by karankartikeya** [reply]
Hey @samus-aran,
Thanks for all the information, but can you please tell what are the steps after the application submission as I was exploring to apply for my SaaS startup as well.
